frontend/src/assets/manual/en-GB/query.md

```diff
@@ -27,6 +27,18 @@ By default the search will combine all terms using `OR`. This means that when yo
 ### Be Careful with Spaces
 Adding or removing a space can change the results of your query. For example search for `+- term` is different than searching for `+-term`. It might be necessary to escape a space (also by placing a `\` in front of it).
 
+### Advanced options to search for combinations of words
```

Comment:
This looks clear! It seems to contradict line 21, though. I think that one needs to be cleared up.